WebRoot/js/zimbraAdmin/common/ZaModel.js

```diff
@@ -437,6 +437,133 @@ ZaModel.getMailPollingIntervalChoices = function () {
 
 ZaModel.MAIL_POLLING_INTERVAL_CHOICES = ZaModel.getMailPollingIntervalChoices;
 
+ZaModel.getMailRecallChoices = function () {
+    return [
+        {
+            value: "1",
+            label: "1 " + ZaMsg.LBL_minute
```

Comment:
we should use ZmMsg instead of ZaMsg.We already have some pre defined values such as ZmMsg.minutes and ZmMsg.minute

Reply:
As part of research, figured out we should be using ZaMsg in order to add labels for admin console.So we can go ahead with ZaMsg labels.
Thanks